resources/lib/listing/listitem.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, title, id, item_type, thumbnail="", geo_restricted=False, inf
         self.item_type = item_type
         self.info = info
         self.fanart = fanart
+        self.season_title = season_title
```

Comment:
After further thinking I think you should only add the `season_title` atttribute to VideoItem and not the generic PlayItem as ShowItem doesn't need it.

Reply:
Updated